From c8b7cc5d4de0387c8edf51da6a85ab05ca710236 Mon Sep 17 00:00:00 2001
From: Kaarle Ritvanen <kaarle.ritvanen@datakunkku.fi>
Date: Wed, 8 Apr 2020 00:18:38 +0300
Subject: [PATCH 3/3] udev-settle: do not require udev

The purpose of this service is to ensure all udev events have been processed.
It is not to be directly added to any runlevel but rather to be required by
other services. This change allows this service as a dependency even when udev
is not used but /dev is populated by another service.

Signed-off-by: Kaarle Ritvanen <kaarle.ritvanen@datakunkku.fi>
---
 init.d/udev-settle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.d/udev-settle b/init.d/udev-settle
index a5ceb93..2b5767e 100644
--- a/init.d/udev-settle
+++ b/init.d/udev-settle
@@ -3,7 +3,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 depend() {
-	need udev
+	need dev
 	after udev-trigger
 	provide dev-settle
 	keyword -lxc -systemd-nspawn -vserver
-- 
2.25.2

